lib/badge-data.spec.js

```diff
@@ -12,65 +13,67 @@ const {
 } = require('./badge-data');
 
 describe('Badge data helpers', function() {
-  it('should detect prefixes', function() {
-    assert.equal(hasPrefix('data:image/svg+xml;base64,PHN2ZyB4bWxu', 'data:'), true);
-    assert.equal(hasPrefix('data:foobar', 'data:'), true);
-    assert.equal(hasPrefix('foobar', 'data:'), false);
+  test(hasPrefix, () => {
+    forCases([
```

Comment:
The documentation states that `forCases` "accepts either an array or a series of arguments". Is there any value in wrapping arguments in arrays rather than listing them directly?

Reply:
With trailing commas, they're easier to reorder, copy, and paste, and when you add another one it makes a one-line change.

Those docs are kind of confusing, too. The language makes it sound like you can pass in an array of arguments like `[1, 2, 3]`, though they need to be cases: `[given(1), given(2), given(3)]`.

Reply:
Fair enough, makes sense.

Reply:
They seem harder to read grouped, so I'm removing the `forCases()` call instead.
```js
  test(byPercentage, () => {
    given(-1).expect('red');
    given(0).expect('red');
    given(0.5).expect('yellow');
    given(1).expect('yellow');
    given(50).expect('yellow');
    given(80).expect('yellowgreen');
    given(85).expect('yellowgreen');
    given(90).expect('green');
    given(100).expect('brightgreen');
    given(101).expect('brightgreen');

    forCases([-1, 0, 0.5, 1, 50, 80, 85, 90, 100, 101]
      .map(v => given(v).expect(coveragePercentage(v))))
      .should("return '%s', for parity with coveragePercentage()");
  });
```